NEMO.md

```diff
@@ -0,0 +1,268 @@
+# NEMO v4.0.6 Porting Guide(Cent OS 7.6)
+
+## Introduction
+
+
+
+## Environment Requirements
+
+### Software Requirements
+
+| Item     | Version | Download Address                                             |
+| -------- | ------- | ------------------------------------------------------------ |
+| HDF5     | 1.10.6  | https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.6/src/ |
+| PNETCDF  | 1.12.1  | https://parallel-netcdf.github.io/wiki/Download.html         |
+| NETCDF-C | 4.7.3   | https://github.com/Unidata/netcdf-c/releases/tag/v4.7.3      |
+| NETCDF-F | 4.5.2   | https://github.com/Unidata/netcdf-fortran/releases/tag/v4.5.2 |
+| XIOS     | 2.5     | https://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/branchs/xios-2.5 |
+| NEMO     | 4.0.2   | https://forge.ipsl.jussieu.fr/nemo/svn/NEMO/releases/r4.0/r4.0.6 |
+
+### OS Requirements
+
+| Item   | Version    | How to Obtain                    |
+| ------ | ---------- | -------------------------------- |
+| CentOS | 7.6        | https://www.centos.org/download/ |
+| Kernel | 4.14.0-115 | Included in the OS image.        |
+
+## Configuring the Compilation Environment
+
+### Installing dependencies
+
+
+    yum install time -y
+    yum install curl* -y
+    yum install csh -y
+    yum install zlib* -y
+    yum install perl-lib* -y
+
+### Installing GNU 9.3
+
+
+    yum install -y centos-release-scl
+    yum install -y devtoolset-9-gcc
+    yum install -y devtoolset-9-gcc-c++
+    yum install -y devtoolset-9-binutils
+    scl enable devtoolset-9 bash
+    echo "souce /opt/rh/devtoolset-9/enable" >> /etc/profile
+
+### Installing Open MPI
+
+1. Run the following command to install the system dependency package:
+
+
+    yum install libxml2* systemd-devel.aarch64 numa* -y
+
+2. Run the following commands to install Open MPI:
+
+
+    wget https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.3.tar.gz
+    tar -zxvf openmpi-4.0.3.tar.gz
+    cd openmpi-4.0.3
+    ./configure --prefix=/path/to/OPENMPI --enable-pretty-print-stacktrace --enable-orterun-prefix-by-default  --with-cma --enable-mpi1-compatibility
+    make -j 16
+    make install
+
+3. Configure environment variables:
+
+
+    export PATH=/path/to/OPENMPI/bin:$PATH
+    export LD_LIBRARY_PATH=/path/to/OPENMPI/lib:$LD_LIBRARY_PATH
+
+### Installing HDF5
+
+1. Run the following commands to install HDF5:
+
+
+    wget https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.6/src/hdf5-1.10.6.tar.gz
+    tar -zxvf hdf5-1.10.6.tar.gz
+    cd hdf5-1.10.6
+    mkdir -p /path/to/HDF5
+    ./configure --prefix=/path/to/HDF5 --build=aarch64-unknown-linux-gnu --enable-fortran --enable-static=yes --enable-parallel --enable-shared CC=mpicc CXX=mpicxx FC=mpifort F77=mpifort
+    make -j 16
+    make install
+
+2. Configure environment variables:
+
+
+    export PATH=/path/to/hdf5/bin:$PATH
```

Comment:
s/hdf5/HDF5/
Better: export an env var for this path and reuse it later

Reply:
done

Reply:
done